### PR TITLE
feat: replace grade available response with esg link when esg flag enable

### PR DESCRIPTION
**Pull request by leangseu-edx**
**TL;DR -** replace grade available response with esg link to new page when esg flag enable

JIRA: [AU-530](https://openedx.atlassian.net/browse/AU-530)

**What changed?**

Screen Shot 2022-03-28 at 1 43 34 PM

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

File: openassessment/xblock/test/test_staff_area.py [hunk 1]
```diff
@@ -1291,6 +1292,65 @@ def test_staff_area_student_info__different_team(self, xblock):
...
+        ORA_GRADING_MICROFRONTEND_URL='ora_url'
+    )
+    @patch(
+        'openassessment.xblock.config_mixin.ConfigMixin.is_enhanced_staff_grader_enabled',
+        new_callable=PropertyMock
+    )
+    @scenario('data/staff_grade_scenario.xml', user_id='Bob')
+    def test_staff_area_esg_enabled(self, xblock, mock_esg_flag):
+        """
+        When esg flag is enabled, enhanced_staff_grader_url should be defined as follow.
```
Comment (jansenk) [inline openassessment/xblock/test/test_staff_area.py:1323]:
```suggestion
        When esg flag is enabled, enhanced_staff_grader_url should be "ORA_GRADING_MICROFRONTEND_URL/xblock_id".
```

File: openassessment/xblock/test/test_staff_area.py [hunk 2]
```diff
@@ -1291,6 +1292,65 @@ def test_staff_area_student_info__different_team(self, xblock):
...
+    def test_staff_area_esg_disabled(self, xblock, mock_esg_flag):
+        """
+        When esg flag is disabled, enhanced_staff_grader_url should not be define.
+        """
+        mock_esg_flag.return_value = False
+        _, context = xblock.get_staff_path_and_context()
+
+        self._verify_staff_assessment_context(context, True, 0, 0)
+        mock_esg_flag.assert_called_once()
+        self.assertRaises(KeyError, lambda: context['enhanced_staff_grader_url'])
```
Comment (jansenk) [inline openassessment/xblock/test/test_staff_area.py:1352]:
```suggestion
        self.assertNotIn('enhanced_staff_grader_url', context)
```

**Review by jansenk — CHANGES_REQUESTED** [review]
Some testing changes but the functionality looks good. Haven't manually tested though

**Review by jansenk — APPROVED** [review]
👍
